audio/beets/Portfile

```diff
@@ -0,0 +1,601 @@
+                    port:py${python.version}-requests \
+                    port:py${python.version}-requests-oauthlib \
+                    port:py${python.version}-six \
+                    port:py${python.version}-soco \
+                    port:py${python.version}-unidecode \
+                    port:py${python.version}-xdg \
+                    port:py${python.version}-yaml
+
+    depends_run-append \
+                    port:chromaprint \
+                    port:ffmpeg \
+                    port:flac \
+                    port:mp3val \
+                    port:ImageMagick \
+                    port:ipfs
+
+    post-destroot {
+        xinstall -d ${destroot}${prefix}/share/man/man1
+        xinstall -m 0644 {*}[glob ${worksrcpath}/man/*.1] ${destroot}${prefix}/share/man/man1/
+
+        xinstall -d ${destroot}${prefix}/share/man/man5
+        xinstall -m 0644 {*}[glob ${worksrcpath}/man/*.5] ${destroot}${prefix}/share/man/man5/
+
+        xinstall -d ${destroot}${prefix}/share/${name}
+        xinstall -m 0644 -W ${worksrcpath} README.rst LICENSE ${destroot}${prefix}/share/${name}
+    }
+} else {
+    depends_lib-append \
+                    port:beets
+
+    post-destroot {
+        foreach f {__init__.py __init__.pyc __init__.pyo __pycache__} {
+            set fp "${destroot}${python.pkgd}/beetsplug/$f"
+            if {[file exists ${fp}]} {
+                file delete -force ${fp}
+            }
+        }
+    }
+}
+
+subport beets-alternatives {
+    name            beets-alternatives
+    version         0.10.2.post1
+    revision        0
+
+    license         MIT
+
+    description     beets plugin to manage multiple files
+    long_description ${description}
+
+    homepage        http://www.github.com/geigerzaehler/beets-alternatives
+
+    checksums       rmd160  832db7cf388d766671184b53e9d2a9ce2a578e7e \
+                    sha256  2ad1213b9a395233082062860a8ba165c29a4125624f7cd9a3a749ee15600bd6 \
+                    size    10943
+}
+
+subport beets-amazon {
+    PortGroup       github 1.0
+
+    github.setup    jmwatte beet-amazon da25d7671f3814a8401b004f592ccf0633197edf
+    name            beets-amazon
+    version         20210908
+    revision        0
+
+    # See: https://github.com/jmwatte/beet-amazon/issues/7
+    license         MIT
+
+    description     plugin for beets that lets you search amazon for album-details
+    long_description ${description}
+
+    checksums       rmd160  144a3a4935fa56b30266f6b8233850d2836da406 \
+                    sha256  a6dc5061c9354b8e6fa77a42f7568f07e01ef8011a267547f291aa3a0f7098ca \
+                    size    3532
+
+    depends_lib-append \
+                    port:py${python.version}-bottlenose
+
+    build {}
+
+    destroot {
+        xinstall -d ${destroot}${prefix}/share/${name}
+        xinstall -m 0644 ${worksrcpath}/README.md ${destroot}${prefix}/share/${name}
+
+        xinstall -d ${destroot}${python.pkgd}/beetsplug
```

Comment:
I thought this directory was already made by the `beets` main port, correct? If so, there is no need to do this.

Reply:
@reneeotten this is a wired plugin which hasn't use any way to install it. It is just `.py` file and `Readme`.

Without this line build is failed with:
```
Error: Failed to destroot beets-amazon: xinstall: Unable to create new file for: /opt/local/var/macports/build/_Users_catap_src_macports-ports_audio_beets/beets-amazon/work/destroot/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/beetsplug, No such file o
```

Reply:
Yeah, true... the directory is created by the `beets` port, but you do  need to make if first in the `destroot` here.